File: codes/questao05/stringInvert.js
function inverterString(str) {
  let caracteres = str.split("");
  let inicio = 0;
  let fim = caracteres.length - 1;

  while (inicio < fim) {
    let temp = caracteres[inicio];
    caracteres[inicio] = caracteres[fim];
    caracteres[fim] = temp;

    inicio++;
    fim--;
  }

  return caracteres.join("");
}

let minhaString = "Exemplo";

console.log("String original: " + minhaString);

minhaString = inverterString(minhaString);

console.log("String invertida: " + minhaString);

File: codes/questao05/stringInvert.c
#include <stdio.h>
#include <string.h>

void inverterString(char str[])
{
    int inicio = 0;
    int fim = strlen(str) - 1;

    while (inicio < fim)
    {

        char temp = str[inicio];
        str[inicio] = str[fim];
        str[fim] = temp;

        inicio++;
        fim--;
    }
}

int main()
{
    char minhaString[] = "Exemplo";

    printf("String original: %s\n", minhaString);

    inverterString(minhaString);

    printf("String invertida: %s\n", minhaString);

    return 0;
}